### DISCUSSION TITLE
You saved me hours of work...

### DISCUSSION THREAD

**Discussion by DanWearr** [mdbootstrap/TW-Elements#1535] [Kind words & general feedback]
I just wanted to take a moment to thank you for your awesome open-source collection of components for Tailwind CSS.

To be honest, I'm still not entirely sold on Tailwind and I'm only using it because I have to for a project. But I'm slowly getting accustomed to it and starting to see the appeal the utility-first approach.

However, there are still so many things that annoy me about it, especially when it comes to creating complex layouts. That's where Tailwind Elements comes in. It already saved me so much time by providing drafts that I can easily drop into my project & customize.

So, thank you again to the creators of Tailwind Elements for putting in the time and effort to create such an awesome toolkit. 
It has already made my life a lot easier, so I can't wait for the stable release!

**Comment by filipkappa** [comment]
Awesome feedback! Thanks for putting your time into it :D 

We hope to save you even more hours (maybe days? 😛 ) of work in the future!
